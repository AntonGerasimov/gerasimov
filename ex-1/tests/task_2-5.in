2147483612 10020
