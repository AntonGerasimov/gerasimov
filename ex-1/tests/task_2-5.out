2147483647 + 9985
